python-sdk/src/astro/files/locations/local.py

```diff
@@ -33,3 +33,19 @@ def size(self) -> int:
         """
         path = pathlib.Path(self.path)
         return os.path.getsize(path)
+
+    @property
+    def openlineage_dataset_namespace(self) -> str:
+        """
+        Returns the open lineage dataset namespace as per
+        https://github.com/OpenLineage/OpenLineage/blob/main/spec/Naming.md
+        """
+        return ""
```

Comment:
the empty namespace is ok?

Reply:
I have changed it to the filename in [commit](https://github.com/astronomer/astro-sdk/pull/1025/commits/a964edfbc763fd61cc615004b657aaf55b4e0d52)

I checked with Maciej from the open lineage team.